source/User_Guide/Settings/two_factor_authentication.md

```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
 You may choose from two different verification methods when using two-factor authentication:
 
 1. Generating a verification code in the [Authy App](https://www.authy.com/app/mobile/).
```

Comment:
This is a fragment:

Generate a verification code with the [Authy App](https://www.authy.com/app/mobile/).